docs/strings.rst

```diff
@@ -148,6 +148,28 @@ this and "pretend" to support it, h5py will raise an error when attempting
 to create datasets or attributes of this type.
 
 
+Handling of iterables of strings as attributes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you set an attribute equal to a Python list or tuple of unicode strings,
+such as the following:
+
+    >>> f.attrs['x'] = (u'a', u'b') # PY2
+    >>> f.attrs['x'] = ('a', 'b') # PY3
```

Comment:
The `u` prefix is valid in Python 3 as well, if you want to simplify the example.